pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>web.accounts</artifactId>
+    <version>0.1.0</version>
+    <packaging>jar</packaging>
+
+    <name>web.accounts</name>
+    <description>Web service to handle company account data</description>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.3.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>1.8</java.version>
+
+        <lombok.version>1.18.0</lombok.version>
+        <thymeleaf-extras.version>3.0.1.RELEASE</thymeleaf-extras.version>
+        <structured-logging.version>1.2.0-rc1</structured-logging.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
This is brought in with `spring-boot-starter-parent`.

Reply:
Need to do some more fiddling with dependency management for this to be true. Will leave it as-is for now.

Reply:
👍